examples/TLS_DenseSkOp.cpp

```diff
@@ -0,0 +1,164 @@
+
+    // Initialize noisy gaussian data
+    init_noisy_data(m, n, d, AB);
+
+    // Define properties of the sketching operator
+
+    // Initialize seed for random number generation
+    uint32_t seed = 0;
+
+    // Define the dense distribution that the sketching operator will sample from
+    /* Additional dense distributions: RandBLAS::DenseDistName::Uniform - entries are iid drawn uniform [-1,1]
+     *                                 RandBLAS::DenseDistName::BlackBox - entires are user provided through a buffer
+    */
+    auto time_constructsketch1 = high_resolution_clock::now();
+    RandBLAS::DenseDistName dn = RandBLAS::DenseDistName::Gaussian;
+    
+    // Initialize dense distribution struct for the sketching operator
+    RandBLAS::DenseDist Dist(sk_dim,   // Number of rows of the sketching operator 
+                             m,        // Number of columns of the sketching operator
+                             dn);     // Distribution of the entires
+
+    //Construct the dense sketching operator
+    RandBLAS::DenseSkOp<double> S(Dist, seed);  
+    RandBLAS::fill_dense(S);
+    auto time_constructsketch2 = high_resolution_clock::now();
+
+    // Sketch AB
+    // SAB = alpha * \op(S) * \op(AB) + beta * SAB
+    auto time_sketch1 = high_resolution_clock::now();
+    RandBLAS::sketch_general<double>(
+            blas::Layout::ColMajor,    // Matrix storage layout of AB and SAB
+            blas::Op::NoTrans,         // NoTrans => \op(S) = S, Trans => \op(S) = S^T
+            blas::Op::NoTrans,         // NoTrans => \op(AB) = AB, Trans => \op(AB) = AB^T
+            sk_dim,                    // Number of rows of S and SAB
+            n+d,                       // Number of columns of AB and SAB
+            m,                         // Number of rows of AB and columns of S
+            1,                         // Scalar alpha - if alpha is zero AB is not accessed
+            S,                         // A DenseSkOp or SparseSkOp sketching operator
+            0,                         // Row offset of S
+            0,                         // Column offset of S
+            AB,                        // Matrix to be sketched
+            m,                         // Leading dimension of AB
+            0,                         // Scalar beta - if beta is zero SAB is not accessed
+            SAB,                       // Sketched matrix SAB
+            sk_dim                     // Leading dimension of SAB
+    );
+    auto time_sketch2 = high_resolution_clock::now();
+
+    // Perform SVD operation on SAB
+    auto time_TLS1 = high_resolution_clock::now();
+    lapack::gesdd(lapack::Job::AllVec, sk_dim, (n+d), SAB, sk_dim, svals, U, sk_dim, VT, n+d);
+            
+    for (int i = 0; i < n; i++) {
+        X[i] = VT[(n+d-1) + i*(n+d)]; // Take the right n by 1 block of V
+    }
+
+    // Scale X by the inverse of the 1 by 1 bottom right block of V
+    blas::scal(n, -1/VT[(n+d)*(n+d)-1], X, 1); 
```

Comment:
This implementation is only valid when ``d = 1``. With that in mind it's probably best to eliminate the symbol ``d`` entirely and just replace it directly with 1 when needed.